ArtOfIllusion/src/artofillusion/SplineMeshEditorWindow.java

```diff
@@ -758,8 +758,7 @@ void subdivideCommand()
     param = new double [usize+usplitcount][vsize][numParam];
     for (i = 0; i < param.length; i++)
       for (j = 0; j < param[i].length; j++)
-        for (int k = 0; k < param[i][j].length; k++)
```

Comment:
As a bit of a smoke test, I chose to benchmark this particular method. I used `System.nanoTime()` to time the method `subdivideCommand()` from method entrance to end of the block. Then I loaded up a file that I have with a nice, 5X9 vertice spline mesh, and started subdividing. Timing is pretty comparable. In the early runs, 1st and 2nd subdivision, the times are within about 2%, with a slight edge to the `System.arraycopy()` version. As expected, arraycopy strengthens its lead in later subdivisions with larger arrays to copy. After about 5 subdivisions, method time was increasing radically in both cases, but the bottleneck seemed to be with rendering the subdivided mesh, not in the subdivision. The version with the `System.arrayCopy()` was able to recover a little better. (At least I was able to do a proper shutdown, instead of force-killing the process)

I'm sold - I don't see any reason not to merge this one.